cmake/faceliftMacros.cmake

```diff
@@ -359,6 +361,8 @@ macro(_facelift_parse_target_arguments additionalOptions additionalOneValueArgs
     set(oneValueArgs ${additionalOneValueArgs})
     set(multiValueArgs PRIVATE_DEFINITIONS PUBLIC_DEFINITIONS PROPERTIES COMPILE_OPTIONS
         HEADERS HEADERS_GLOB HEADERS_GLOB_RECURSE
+        HEADERS_NO_MOC
+        HEADERS_NO_MOC_FILE
```

Comment:
not needed

Reply:
needed for lists created by applications that can use facelift's facelift_add_library